tests/e2e/util.go

```diff
@@ -2269,3 +2272,41 @@ func getDefaultDatastore(ctx context.Context) *object.Datastore {
 
 	return defaultDatastore
 }
+
+//setClusterDistribution returns boolean
```

Comment:
Can you change the comment to `setClusterDistribution sets the input cluster-distribution in vsphere-config-secret`